src/autoscaler/metricsforwarder/cmd/metricsforwarder/main.go

```diff
@@ -64,13 +64,14 @@ func main() {
 	}
 	defer policyDB.Close()
 
-	pm := cred_helper.PluginManager{}
+	pm := cred_helper.New()
+	defer pm.Kill()
```

Comment:
hmmm ... with the terminate does the plugin close the db connection it creates ?
Actually we should be doing this in the defer too.